src/java/org/apache/cassandra/tools/nodetool/Repair.java

```diff
@@ -105,6 +105,9 @@ public class Repair extends NodeToolCmd
     @Option(title = "paxos-only", name = {"-paxos-only", "--paxos-only"}, description = "If the --paxos-only flag is included, no table data is repaired, only paxos operations..")
     private boolean paxosOnly = false;
 
+    @Option(title = "accord-only", name = {"-accord-only", "--accord-only"}, description = "If the --accord-only flag is included, no table data is repaired, only accord operations..")
+    private boolean accordOnly = false;
```

Comment:
Last we discussed this you actually pushed to not expose this to users in repair and I still agree with that. An accord-only repair isn't something useful to end users and we added a separate finish migration step to the JMX interface to handle invoking the Accord only repair.

We definitely should as part of repair do the Accord repairs to help maintain the contract people rely on where the repaired set is safe to read at CL=1.

Reply:
I think my objection was exposing the initial iteration of accord repair as a repair, since it didn't sync data on disk between all nodes. An accord _only_ repair is a little different, it's basically the accord version of a paxos-only repair. Both paxos and accord only repairs only complete pending operations and do not sync data.

I'm also don't agree that accord-only repairs wouldn't be useful to end users. For instance, one of the use cases of paxos repairs is to establish a lower bound for applied paxos operations so that data can be exported to an ldap system for processing. Having said that, the accord repair is not on par with paxos repair since it issues a barrier, but does not wait for it to be applied on all nodes, and I think that's something I should add before committing